collect/sentinel/sentinel-proxy/patches/0001-Fix-compilation-with-OpenWRT-master-23.05.patch

```diff
@@ -0,0 +1,28 @@
+From 5da0dfe0b3a7ad0488ddf8550f15e6b0832fba59 Mon Sep 17 00:00:00 2001
+From: Magnus Kessler <Magnus.Kessler@gmx.net>
+Date: Wed, 2 Aug 2023 14:39:47 +0200
+Subject: [PATCH] Fix compilation with OpenWRT master / 23.05
+
+Recent versions of the msgpack C-library are installed as msgpack-c.
```

Comment:
As well here - patch should be submitted to appropriate repository, in this case, sentinel-proxy.